### PR TITLE
Settings Fixes

### PR DESCRIPTION
**Pull request by evantahler**
* Enforce that Numeric Settings cannot be negative
* Remove un-used "License" setting
 
### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

>

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Review by krishnaglick — APPROVED** [review]
This looks good. Only comment is we might want to use `<input type="number" min=0 />` for em.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

**Comment by evantahler** [comment]
> min=0

Nice! Added in https://github.com/grouparoo/grouparoo/pull/2424/commits/45a6786492e28526f30b57d0deeab5056354a4b0

**Review by krishnaglick — APPROVED** [review]
👍 👍

File: ui/ui-components/components/settings/settingCard.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default function SettingCard({
                     style={{ width: "100%" }}
                     required
                     type="number"
+                    min="0"
```
Comment (bleonard) [inline ui/ui-components/components/settings/settingCard.tsx:55]: It's interesting that some of the minimums might actually be 1
I'm not sure 0 makes sense in this case.
Not a high priority, though.
Screen Shot 2021-10-14 at 5 00 38 PM

Comment (bleonard) [reply ui/ui-components/components/settings/settingCard.tsx:55]: Actually, maybe that's ok because it's number of "retries" not total number of tries.

**Pull request merged**